test/qunit/model/FHIRModel.unit.js

```diff
@@ -1265,4 +1264,44 @@ sap.ui.define([
 		assert.strictEqual(oValueSet.url, "https://standards.digital.health.nz/fhir/ValueSet/sact-location-medication-collection-code", "ValueSet Search Response is saved in the model data without throwing an error");
 	});
 
+	QUnit.test("complex filter test with multiple group filters", function (assert) {
+		var oFhirModel = createModel({ simpleFiltering: false });
```

Comment:
could you format the test to more visible logical blocks e.g. inserting empty lines

Reply:
done

Reply:
done